docs/LV_client/simulator/mono_send_step.md

```diff
@@ -0,0 +1,33 @@
+# mono_send_step.vi
+
+<p class="img_container">
+<img class="lg_img" src="../mono_send_step.png"/>
+</p>
+
+### Description
+
+Send the command to step the simulation for a given amount. Specify the desired amount in the message string.
+
+For technical support contact us at <b>support@monodrive.io</b> 
+
+### Inputs
+
+- **Command_ID:**  Command to move one step in simulation.  Valid message
+**REPLAY_StepSimulationCommand_ID**
+ 
+
+- **message:**  A string in the following format where the number specifies
+how many steps to move in simulation.   
+
+```
+{"amount":1}
+```   
+ 
```

Comment:
May want to try indenting this to see if it shows up inline with the bulleted list